core/src/main/kotlin/org/evomaster/core/problem/graphql/GqlConst.kt

```diff
@@ -49,7 +49,10 @@ object GqlConst {
     const val ENUM = "enum"
     const val LIST = "list"
     const val INPUT_OBJECT = "input_object"
-    /**/
+    /*
+    *Those are entry points of GraphQL query and mutation
+    * Queries and mutations names are defined by developers, so they can have different names
+    * eg: query, rootquery ...  */
     const val QUERY = "query"
     const val QUERY_TYPE = "querytype"
     const val ROOT = "root"
```

Comment:
names could be anything. cannot hardcode here.
need to check based on actual type in schema, NOT the name.
see https://graphql.org/learn/schema/

Reply:
create a test in which we have something like `foo: Query`

Reply:
Done